app/config/routing.yml

```diff
@@ -194,6 +194,16 @@ interviews:
     defaults:
          _controller: AppBundle:Interviews:list
 
+job-advert:
+    path: /job-adverts/{id}/{slug}
+    defaults:
+         _controller: AppBundle:JobAdverts:jobAdvert
```

Comment:
Should make `slug` optional, it's already handled in the controller (`checkSlug()` gives redirects automatically if the slug changes etc, plus it easier to type without it).